content/doc/book/pipeline/multibranch.adoc

```diff
@@ -0,0 +1,107 @@
+---
+layout: section
+---
+:notitle:
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:imagesdir: /doc/book/resources/pipeline
+:hide-uri-scheme:
+:toc:
+
+= Multibranch Pipelines
+
+In the <<jenkinsfile#, previous section>> a `Jenkinsfile` which could be
+checked into source control was implemented. This section covers the concept of
+*Multibranch* Pipelines which build on the `Jenkinsfile` foundation to provide
```

Comment:
@scribblingfox - Nice! I knew there was a way to nuke that "could be" but I couldn't quite get there. 
I've filed an issue to get this stuff into the style guide where possible. 
https://issues.jenkins-ci.org/browse/WEBSITE-268